fetch_data.sh

```diff
@@ -29,6 +29,10 @@ cd ../../../
 
 # Get codelists for versions v1.x and v2.x of the IATI Standard
 cd data
+# Switch to python 3 environment
+if [ ! -d pyenv ]; then
+	python3 -m venv pyenv
+source pyenv/bin/activate
```

Comment:
This `if` needs a corresponding `fi`, otherwise the following error shows in the logs:

```
./fetch_data.sh: line 61: syntax error: unexpected end of file
```